server/database/queries/captain/getCaptains.js

```diff
@@ -0,0 +1,7 @@
+const connect = require('../../config/dbConnection');
+
+const getCaptains = () => {
+  const sql = 'SELECT * FROM tuser where i_type=1';
```

Comment:
@Jamalat-shamallakh I find it helpful if there are hard-coded types, to put them in a variable somewhere so that they have a name.
For example, in another file, you could have
```
export const ADMIN_USER_TYPE = 1;
export const NON_ADMIN_USER_TYPE = 2;
```
and so on
And then here, you could use that:
`const sql = 'SELECT * FROM tuser where i_type=$1'` and the values `[ ADMIN_USER_TYPE ]`.
That makes life easier because you don't all have to remember which magic number means which user type (it makes things less bug prone).
It's not an urgent change that you have to make now, but maybe an idea for the next app you make.